commands/update/nodepool/command.go

```diff
@@ -194,10 +196,14 @@ func updateNodePool(args Arguments) (*result, error) {
 			modifyRequestBody.Name = args.Name
 			updated = true
 		}
-		if args.ScalingMin != 0 && args.ScalingMin != existingNP.Payload.Scaling.Min ||
-			args.ScalingMax != 0 && args.ScalingMax != existingNP.Payload.Scaling.Max {
+
+		//Check for non-input from user, set value to current min
+		if !args.ScalingMinSet {
+			args.ScalingMin = *existingNP.Payload.Scaling.Min
+		}
+		if (args.ScalingMin >= 0 && args.ScalingMin != *existingNP.Payload.Scaling.Min) || (args.ScalingMax >= 0 && args.ScalingMax != existingNP.Payload.Scaling.Max) {
```

Comment:
Is it always safe to assume that `existingNP.Payload.Scaling.Min` is not `nil`? Trying to dereference it with `*existingNP.Payload.Scaling.Min` will throw an error if the pointer is nil.